pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go

```diff
@@ -473,10 +466,7 @@ func (gqm *GroupQuotaManager) GetAllQuotaNames() map[string]struct{} {
 	return quotaInfoMap
 }
 
-func (gqm *GroupQuotaManager) UpdatePodRequest(quotaName string, oldPod, newPod *v1.Pod) {
```

Comment:
why? I think we need hierarchyUpdateLock to avoid update tree topo concurrently

Reply:
the hierarchyUpdateLock is locked by the functions which call this function.

Reply:
as discuessed, we don't let hierarchyUpdateLock out of gqm.

Reply:
fixed